README.md

```diff
@@ -39,31 +39,31 @@ $ brew remove eosio
 ```
 #### Ubuntu 18.04 Debian Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc1/eosio_1.5.0-rc1-1-ubuntu-18.04_amd64.deb
-$ sudo apt install ./eosio_1.5.0-rc1-1-ubuntu-18.04_amd64.deb
+$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc2/eosio_1.5.0-rc2-1-ubuntu-18.04_amd64.deb
+$ sudo apt install ./eosio_1.5.0-rc2-1-ubuntu-18.04_amd64.deb
 ```
 #### Ubuntu 16.04 Debian Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc1/eosio_1.5.0-rc1-1-ubuntu-16.04_amd64.deb
-$ sudo apt install ./eosio_1.5.0-rc1-1-ubuntu-16.04_amd64.deb
+$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc2/eosio_1.5.0-rc2-1-ubuntu-16.04_amd64.deb
+$ sudo apt install ./eosio_1.5.0-rc2-1-ubuntu-16.04_amd64.deb
 ```
 #### Debian Package Uninstall
 ```sh
 $ sudo apt remove eosio
 ```
 #### Centos RPM Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc1/eosio-1.5.0-rc1-1.el7.x86_64.rpm
-$ sudo yum install ./eosio-1.5.0-rc1-1.el7.x86_64.rpm
+$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc2/eosio-1.5.0-rc2-1.el7.x86_64.rpm
+$ sudo yum install ./eosio-1.5.0-rc2-1.el7.x86_64.rpm
```

Comment:
since we are now passing in `rc2` as the release the trailing 1 should be removed (ie. `eosio-1.5.0-rc2.el7.x86_64.rpm`). This is for both rpms but **NOT** the debs